docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -415,6 +415,8 @@ pushed down to S3 Select. Changes in the Hive connector :ref:`performance tuning
 configuration properties <hive-performance-tuning-configuration>` are likely to impact
 S3 Select pushdown performance.
 
+S3 Select can be enabled for CSV data using the ``hive.experimental-pushdown-enabled`` configuration
```

Comment:
We should say TEXTFILE here since CSV is actually a different Hive format and is not supported. The fact that S3 calls this CSV is an implementation detail.